docker/README.md

```diff
@@ -83,6 +83,7 @@ deploy your subgraph to the running Graph Node.
   
 We do not currently build native images for Macbook M1, which can lead to processes being killed due to out-of-memory errors (code 137). Based on the example `docker-compose.yml` is possible to rebuild the image for your M1 by running the following, then running `docker-compose up` as normal:
  
+It is important to increase memory limits for the docker engine running on your machine. Otherwise docker build command will fail due to out of memory error. To do that, open docker-desktop and go to Resources/advanced/memory
```

Comment:
Move this below the code block, and use `> **Important** ...` to render it a little bit better.

```
...

# Tag the newly created image
docker tag graph-node graphprotocol/graph-node:latest
`` `

> **Important** <Your message here> 
```

And please the sentence with an ending dot.

Reply:
thanks for review. Done